test/unit-tests/mobile/devices-service.ts

```diff
@@ -607,7 +600,6 @@ describe("devicesService", () => {
 				assert.deepEqual(counter, 1, "The action must be executed on only one device.");
 				counter = 0;
 				androidDeviceDiscovery.emit("deviceLost", tempDevice);
-				await assertAndroidEmulatorIsStarted({ platform: "android" });
```

Comment:
this is the second time the `devicesService.initialize()` is called, and it's expected to not start emulator